### ISSUE TITLE
Wrong calculation involving ZETA (aka DELTAS)

### ISSUE THREAD

**Issue by rdemaria** [SixTrack/SixTrack#1022]
In few places the relation between zeta and sigma is implemented wrongly.

The correct definition is
```
rvv = beta/beta0
sigma= (s - beta0 c t)
zeta=DELTAS=beta/beta0 (s - beta0 c t) = sigma / rvv
```
Wrong places:
```
\zeta=r_v\sigma
chConventions.tex:39:  $\zeta$    & n/a                & [mm]         & $r_v\sigma$ 
chExtraOutput.tex:57:    \texttt{zeta}   & n/a                & [mm]         & $\zeta=r_v\sigma$ 
sixda.f90:764:!FOX  DELTAS=SIGMDA*RV ;
mod_dist.f90:1804:    sigmv(1:napx) = dist_partCol5(1:napx)/rvv(1:napx)
```
```

**Comment by vkbo** [comment]
´´´
zeta=DELTAS=beta/beta0 (s - beta0 c t) = sigma / rvv
´´´
What? How is that *not* rvv*sigma? If sigma is below the line, the write that so there is no ambiguity.

I'm on sick leave until the 9th.

**Comment by vkbo** [comment]
Wait, you changed the issue after the email was sent and flipped the rvv. Then it's fine. Anyway, couldn't you have fixed it when when you went through the code?

**Comment by vkbo** [comment]
Wait, you changed the issue after the email was sent and flipped the rvv to beta0/beta. Then your text makes sense. i was responding from the email.

When I wrote the DIST block I just followed what the table said, and if it was wrong, so is the code. The table in the Conventions chapter is just a copy/paste of the one in DUMP.

Anyway, couldn't you have fixed it when when you went through the code? It shouldn't change more than a couple of tests for the DIST block, unless fixing the whole FOX part is a major thing.

**Comment by rdemaria** [comment]
Thanks a lot for the feedback! Just wanted to see if I missed something and other implications. I wait the feedback from Tobias before fixing...

**Comment by tpersson** [comment]
I agree that the first lines are wrong but the following line I am not sure about:
??? six_fox.f90:163:!FOX  DELTAS=SIGM1 ; I think it might be redundant since DELTAS is only interesting in 6D

**Comment by vkbo** [comment]
I've added a PR #1027 addressing the DIST block and the manual. The FOX part, I leave to the two of you to figure out :)
